### PR TITLE
use logger instead of print

### PR DESCRIPTION
**Pull request by asanin-epfl**
Implementation of #11

### CODE REVIEW

**Comment by asanin-epfl** [comment]
@orena1 please review

File: neuron_reduce/subtree_reductor_func.py [hunk 1]
```diff
@@ -343,9 +345,9 @@ def copy_dendritic_mech(original_seg_to_reduced_seg,
         all_segments.extend(list(bas))
 
     if len(all_segments) != len(reduced_seg_to_original_seg):
-        print('There is no segment to segment copy, it means that some segments in the reduced '
-              'model did not receive channels from the original cell')
-        print('trying to compensate by copying channels from neighboring segments')
+        logger.info('There is no segment to segment copy, it means that some segments in the'
```
Comment (orena1) [inline neuron_reduce/subtree_reductor_func.py]: Can you change this to  warning?

File: neuron_reduce/subtree_reductor_func.py [hunk 2]
```diff
@@ -483,14 +485,14 @@ def synapse_properties_match(synapse, PP, PP_params_dict):
...
+        logger.debug("loading template '%s'" % model_obj_name)
         if model_filename == 'model.hoc':
-            print("loading default reduced model")
+            logger.debug("loading default reduced model")
             load_default_model()
         else:
             h.load_file(model_filename)
     else:
-        print("WARNING The template '" + model_obj_name + "' is already defined... not loading.")
+        logger.warning("The template '%s' is already defined... not loading." % model_obj_name)
```
Comment (orena1) [inline neuron_reduce/subtree_reductor_func.py]: Sorry, this could be info... It does not need to be warning.

**Review by orena1 — CHANGES_REQUESTED** [review]
Great thanks

**Comment by asanin-epfl** [comment]
Done. Please do a squash and merge, so it will be a single commit.

**Comment by orena1** [comment]
Thanks!

**Pull request merged**